examples/README.md

```diff
@@ -8,6 +8,16 @@ In this folder, you can find examples where the Portals functions are used.
 
 * **put_to_self** : show the sending of a message to oneself and handle completion events.
 
-* **hello** : send an hello world from a client to a server (physical addressing). See the source files for a detailed usage.
-    - client : send a message and receive it's ack. Usage: `client <server nid> <message to send>`
-    - server : receive the message and read it. Usage `server [number of messages to wait]`
+* **hello** : show the connection between a transmiter and a receiver.
+    - client : send a message and receive it's ack.
+    - server : receive the message and read it.
+
+* **get_me** : show the get event from a client to a server. 
+    - client2 : try to get messages from the specified server  
+    - server2 : prepare a buffer with messages and put it in the priority list
+
+* **search** : example to use PtlMESearch.
+
+* **get_id** : example to get our different id depending of the logical/physical interface.
+
+* **put_to_self_logical** : show the sending of a message and it's ack using a logical interface. 
```

Comment:
show -> demonstrate
it's -> its

I think we should put this example just after the `put_to_self` in the list